recipes/mgmd.rb

```diff
@@ -1,3 +1,10 @@
+if node['platform_family'].eql?("debian")
+  package  ["libncurses5", "net-tools"] do
+    retries 10
+    retry_delay 30
+  end
+end
```

Comment:
This should be moved in the install.rb recipe here:  https://github.com/logicalclocks/ndb-chef/blob/4f6628f2e9ac69f60ac695a9c6cc5bd0ad02c483/recipes/install.rb#L165 where we install the ncurses library for centos/rhel. 

The reason is there is because the ncurses library is needed by the datanode and mysql as well. It worked for you because you run everything on a single VM, but this PR won't work if you have multiple nodes.

Additionally the `net-tools` package is not needed as `netstat` is deprecated in favor of `ss` which is already installed on the machine.